package.json

```diff
@@ -1,27 +1,30 @@
 {
-  "name": "preact-fluid",
+  "name": "preact-fluid-test",
   "version": "0.5.7",
   "description": "",
-  "main": "src/index.js",
+  "main": "lib/index.js",
   "scripts": {
     "start": "npm run styleguide",
     "generate-component": "plop component",
     "test": "npm run prettier-check && preact test",
     "gh-pages": "./node_modules/.bin/gh-pages -d styleguide",
     "styleguide": "styleguidist server",
     "styleguide-build": "NODE_ENV=production styleguidist build",
+    "build": "babel src --out-dir lib  --source-maps true",
+    "prebuild": "rimraf ./lib",
     "precommit": "lint-staged",
-    "prettier":
-    "prettier --write src/**/*.js",
-    "prettier-check":
-    "prettier --list-different src/**/*.js",
+    "prettier": "prettier --write src/**/*.js",
+    "prettier-check": "prettier --list-different src/**/*.js",
     "publish-docs": "npm run styleguide-build && npm run gh-pages",
-    "publish-patch": "npm version patch && npm publish",
-    "publish-minor": "npm version minor && npm publish",
-    "publish-major": "npm version major && npm publish"
+    "publish-patch": "npm run build && npm version patch && npm publish",
+    "publish-minor": "npm run build && npm version minor && npm publish",
+    "publish-major": "npm run build && npm version major && npm publish"
   },
   "lint-staged": {
-    "*.{js}": ["prettier --write src/**/*.js", "git add"]
+    "*.{js}": [
+      "prettier --write",
```

Comment:
with lint-staged, whatever is staged on git will get appended to the command.
`prettier --write [GIT_STAGE_FILES]`

Reply:
that's right